packages/multiselect/src/react/__specs__/index.spec.tsx

```diff
@@ -33,6 +45,97 @@ describe('MultiSelectField', () => {
       expect(results).toHaveNoViolations()
     })
   })
+
+  describe('Basic story', () => {
+    const { Basic } = stories
+
+    it('should focus input when label clicked', () => {
+      render(<Basic {...(Basic.args as any)} />)
```

Comment:
Why the cast?

Reply:
storybook typings don't infer all the types. i figured since it was a test and the stories are typed this was an ok recast